Walkthrough Guide/06 Functions and Cognitive Services/README.md

```diff
@@ -39,29 +39,29 @@ Once the deployment is succeeded, you can navigate to the resource and access th
 
-This behaviour allows Microsoft to offer a [**very attractive pricing model**](https://azure.microsoft.com/en-us/pricing/details/functions/) where you only pay for pure execution time of an Azure Function. That means you only pay and Azure Function when it is actually used. If you write code that never gets executed, it won't cost you anything! The ultimate idea of cloud computing! Event better, [the first 1 million executions or 400000 GB-s are free](https://azure.microsoft.com/en-us/pricing/details/functions/)!
+This behaviour allows Microsoft to offer a [**very attractive pricing model**](https://azure.microsoft.com/en-us/pricing/details/functions/) where you only pay for pure execution time of an Azure Function. That means you only pay for an Azure Function when it is actually used. If you write code that never gets executed, it won't cost you anything! The ultimate idea of cloud computing! Even better, [the first 1 million executions or 400000 GB-s are free](https://azure.microsoft.com/en-us/pricing/details/functions/)!
 
 > **Hint:** Azure Functions are the ideal service to extend existing large backend architectures with additional functionality or to process data in the cloud. The latter is exactly what we need to do when resizing images. 
 
-Whenever a user uploads an image, he should get immediate feedback and should not have to wait for the Cognitive Services. Once the image gets dropped to the Blob Storage, the Function awakes and calls the Cognitive Service API to resize it in a smart way in the background. Next time a user fetches images, he will receive the resized versions.
+Whenever a user uploads an image, they should get immediate feedback and should not have to wait for the Cognitive Services. Once the image gets dropped to the Blob Storage, the Function awakes and calls the Cognitive Service API to resize it in a smart way in the background. Next time a user fetches images, they will receive the resized versions.
 
-We have already prepared an Azure Function so we don't need to start from scratch! In the repository, there is an Azure Function called [`ResizeImage.cs`](/Backend/Functions/ResizeImage.cs) that contains the code for our scenario.
+We have already prepared an Azure Function so we don't need to start from scratch! In the repository, there is an Azure Function called [`ResizeImage.cs`](/Backend/Functions/ResizeImage.cs) that contains the code for our scenario:
 
 1. Get triggered by a Storage Queue message
-1. Take an image from Azure Blob Storage
-1. Upload it to the Cognitive Services Computer Vision API
-1. Write the resized images back to Azure Blob Storage
-1. Update the Cosmos DB entry
+2. Take an image from Azure Blob Storage
```

Comment:
```suggestion
1. Take an image from Azure Blob Storage
```

Please don't update the Markdown lists. Beginning every line with `1.` is totally fine in Markdown and prevents that you have to adjust the list every time you remove or add an item.